checks.d/openstack.py

```diff
@@ -512,3 +650,57 @@ def check(self, instance):
         except IncompleteConfig:
             self.warning("Configuration Incomplete! Check your openstack.yaml file")
             return
+
+
+    #### Local Info accessors
+    def get_local_hypervisor(self):
+        # Look up hypervisors available filtered by my hostname
+        host = self.get_my_hostname()
+        hyp = self.get_all_hypervisor_ids(filter_by_host=host)
+        return hyp[0]
+
+    def get_local_project(self):
```

Comment:
Ah, I see, that makes sense. I wonder though if it is appropriate to ignore the `check_all_projects` flag in this case. 

For example, in our environment, I plan on running this integration using the `local_only: true` flag on each of the hypervisors. But, because of the way the auth works, I would still want to have a way to gather the stats for each project. In that case, I would most likely want to disable the project check on the individual hypervisors. I would then enable *only* the project check on the controller nodes (with `local_only: true` and `check_all_projects: true` and the rest of the checks disabled).

Not sure if that'd be a common use case or even the most appropriate way to set it up.